kitsune/users/jinja2/users/questions.html

```diff
@@ -0,0 +1,25 @@
+{% extends "users/base_contributions.html" %}
+{% set title = _("{name} | Questions")|f(name=display_name(profile.user)) %}
+{% set canonical_url = canonicalize(viewname="users.questions", username=profile.user.username) %}
+{% set active = "user-questions" %}
+
+{% block contributions %}
+  <h2 class="sumo-callout-heading">
```

Comment:
I would go for `sumo-page-subheading` but in any case documents, questions and answers should use the same style